jenkins/ci.suse.de/templates/cloud-mkcloud-job-monasca-template.yaml

```diff
@@ -0,0 +1,28 @@
+- job-template:
+    name: 'cloud-mkcloud{version}-job-monasca-{arch}'
+    node: cloud-trigger
+    disabled: '{obj:disabled}'
+
+    triggers:
+      - timed: 'H H * * *'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 7
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            TESTHEAD=1
+            cloudsource=develcloud{version}
+            nodenumber=4
+            nodenumbercompute=3
```

Comment:
Why so many compute nodes? Also, for the proposal to automatically do its thing we'll need at least one designated `monitoring` node. `mkcloud` will need a way to know it should create one, i.e. we should include a `nodenumbermonitoring` parameter for that.